third-party/chpl-venv/README.md

```diff
@@ -76,6 +80,14 @@ A YAML parser and emitter for Python
 
 **Website**: https://pypi.python.org/pypi/PyYAML
 
+### scspell
+
+A spell checker for source code
+
+**License**: scspell program: GPL 2; wordlists: public domain / assorted: https://github.com/en-wl/wordlist/blob/master/scowl/Copyright
```

Comment:
I don't think scspell has any package dependencies as such -- its dictionary is included directly its repo and package.  The dictionary was assembled from several word lists in the SCOWL project, and is released under that project's licenses.

(The scspell project also includes the licenses:
https://github.com/myint/scspell/blob/master/SCOWL-LICENSE.txt
I can't think now why I linked to the one in the scowl repo.  The scspell one is more likely to reflect the set of licenses at the time the scspell dictionary was asembled.  I can change that.)